Userland/Libraries/LibJIT/Assembler.h

```diff
@@ -115,18 +115,30 @@ struct Assembler {
         SignedGreaterThanOrEqualTo = 0xD,
         SignedLessThan = 0xC,
         SignedLessThanOrEqualTo = 0xE,
+        SignedOverflow = 0x0,
+        NotSignedOverflow = 0x1,
     };
 
     static constexpr u8 encode_reg(Reg reg)
     {
         return to_underlying(reg) & 0x7;
     }
 
-    enum class Patchable {
+    enum class Patchable : u8 {
         Yes,
         No,
     };
 
+    enum class ArithSign : u8 {
+        Positive,
+        Negative
+    };
```

Comment:
Imo this should be closer to the inc/dec  and add/sub helpers,
And should probably be renamed to
```suggestion
    enum class AddOrSub {
        Add,
        Sub
    };
```